analyzer/python/ikos/scan.py

```diff
@@ -657,7 +669,7 @@ def binaries(self):
 def compile_main(mode, argv):
     progname = os.path.basename(argv[0])
 
-    if 'IKOS_SCAN_SERVER' not in os.environ:
+    if 'IKOS_SCAN_SERVER' not in os.environ and 'IKOS_SCAN_NOTIFIER_FILES' not in os.environ:
```

Comment:
This would make ikos-scan fail for all users that didn't define IKOS_SCAN_NOTIFIER_FILES, we probably don't want that?

Reply:
Addressed in 76ed20d